# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/WorkerPrivate.cpp
# Commit: 0c9650a1ac48
# Full Hash: 0c9650a1ac48ca7e8fd2c60f7f498f381d32a591
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-12-15 09:27:59
# Regressor Bug: 1247687
# File Overlap Count: 1
# Description:
#   Bug 1247687 - Use requestCredentials to set secFlags for modules; r=jonco
#   
#   This implements the specified request credentials behavior from the spec, and is tested in wpt.
#   
#   Depends on D155566
# ==============================================================================

diff -r e7b103c79b1a -r 0c9650a1ac48 dom/workers/WorkerPrivate.cpp
--- a/dom/workers/WorkerPrivate.cpp	Wed Dec 14 14:55:36 2022 +0000
+++ b/dom/workers/WorkerPrivate.cpp	Wed Dec 14 14:55:37 2022 +0000
@@ -2534,11 +2534,10 @@
     WorkerKind aWorkerKind, const nsAString& aWorkerName,
     const nsACString& aServiceWorkerScope, WorkerLoadInfo* aLoadInfo,
     ErrorResult& aRv, nsString aId) {
-  return WorkerPrivate::Constructor(aCx, aScriptURL, aIsChromeWorker,
-                                    aWorkerKind, RequestCredentials::Omit,
-                                    WorkerType::Classic, aWorkerName,
-                                    aServiceWorkerScope, aLoadInfo, aRv,
-                                    std::move(aId));
+  return WorkerPrivate::Constructor(
+      aCx, aScriptURL, aIsChromeWorker, aWorkerKind, RequestCredentials::Omit,
+      WorkerType::Classic, aWorkerName, aServiceWorkerScope, aLoadInfo, aRv,
+      std::move(aId));
 }
 
 // static
